src/freenet/support/compress/NewLZMACompressor.java

```diff
@@ -66,9 +66,6 @@ public Bucket compress(Bucket data, BucketFactory bf, long maxReadLength, long m
 	public long compress(InputStream is, OutputStream os, long maxReadLength, long maxWriteLength,
 						 final long amountOfDataToCheckCompressionRatio, final int minimumCompressionPercentage)
 			throws IOException, CompressionRatioException {
-		if (amountOfDataToCheckCompressionRatio == Long.MAX_VALUE && minimumCompressionPercentage == 0) {
```

Comment:
?!? that's the code I've commented on in the other PR!

You should always rebase your PRs against next

Reply:
sorry about that. They build on each other, I just split them apart at commit boundaries, but did not amend history to get rid of added-and-later-removed changes.